index.html

```diff
@@ -1004,6 +1006,113 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">protocol_handlers</dfn></code> member is an <a>array</a>
+          of <a>ProtocolHandlerItem</a>s that allows a web application to
+          handle URL protocols.
+        </p>
+        <p class="note">
+          Protocol handlers could, for instance, be used for web app
+          communication where one app directly invokes another and passes data
+          via custom protocol links.
+        </p>
+        <p>
+          How protocol handlers are presented, and how many of them are shown
+          to the user, is at the discretion of the user agent and/or operating
+          system.
+        </p>
+        <p>
+          The steps for <dfn>processing the <code>protocol_handlers</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <a data-cite=
+          "WEBIDL#sequence-type">sequence</a>&lt;<a>ProtocolHandlerItem</a>&gt;
+          <var>protocol_handlers</var> and a <a>URL</a> <var>manifest
+          URL</var>. This algorithm returns a <a data-cite=
+          "WEBIDL#sequence-type">sequence</a>&lt;<a>ProtocolHandlerItem</a>&gt;.
+        </p>
+        <ol>
+          <li>Let <var>processedProtocolHandlers</var> be a new Array object
+          created as if by the expression [].
+          </li>
+          <li><a href="https://infra.spec.whatwg.org/#list-iterate">For each</a> <var>protocol_handler</var> (<a>ProtocolHandlerItem</a>)
```

Comment:
[=list/iterate=]

Use xref to find the best way to refer https://respec.org/xref/?term=iterate

Reply:
Hola Kenny. I'm still quite new to this so sorry for the basic question. 😉

I'm trying to change the IDL stuff to references to the infra stuff. I can't seem to find proper examples of how to do so, I kinda get that I need a data-cite, and trying to do this **<a data-cite="#list-iterate">For each</a>** but it is not working. I also tried **<a data-cite="ref-for-list-iterate⑤">For each</a>** trying to get the id's that they have in the infra spec. Can you guide me how to proceed in this case?

Reply:
https://respec.org/xref/?term=for+each will tell you how :-)

Just use `[=list/for each=]`

Reply:
@diekus, pro tip: in theory, if you need `data-cite="spec#thing"`, there is probably something wrong. Always check https://respec.org/xref/ first. 

If the term is not there, it usually means that there is a spec bug (in our spec). In the rare case where something is missing, you then need to ask the spec that you are trying to reference to export the term by adding `data-export=""` or `export` for BikeShed specs.

Hope that makes sense! 

PS: you can always ping us for help!